nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -332,7 +332,7 @@ public DocumentRef getTopLevelCommentAncestor(CoreSession s, DocumentRef documen
     /**
      * Gets or creates the 'Comments' folder, this folder will be under the document being commented and it contains the
      * whole comments of the first document that we comment.
-     * 
+     *
      * @param session the core session
```

Comment:
the change here is not needed

Reply:
There is actually a trailing space. 
Devs using a configured eclipse will systematically run into that because of automation.
https://doc.nuxeo.com/corg/java-code-style/#no-automatic-formatting-on-save
But I agree this has to do with cleaning

Reply:
yes it should be on cleanup commit and for autosave eclipse you should configure it to not reformat the code otherwise i think you will have some issues on `10.10` and `9.10` where we dont cleanup

Reply:
I don't autoformat. Removing trailing spaces is the allowed exception:
https://doc.nuxeo.com/corg/java-code-style/#no-automatic-formatting-on-save
Since it has been handled automatically in the past, the only time I have this issue is with newly created files in your IDE.
I'll move that to cleanup. Otherwise other eclipse users will/may have the same issue.

Reply:
In the end the file remains untouched. Can't commit space char diff on git so I dropped the change